src/renderer/catalogEntityDetails.tsx

```diff
@@ -41,9 +51,9 @@ const catalogEntityDetails = [
             {props.entity.metadata.uid || unknownValue()}
           </DrawerItem>
           <DrawerItem
-            name={strings.catalog.entities.sshKey.details.props.dateCreated()}
+            name={strings.catalog.entities.common.details.props.dateCreated()}
           >
-            {props.entity.spec.created || unknownValue()}
+            {getCreatedValue(props.entity.spec.createdAt) || unknownValue()}
```

Comment:
Since `createdAt` is a required property, this can simply be:

```suggestion
            {getCreatedValue(props.entity.spec.createdAt)}
```

Especially since `getCreatedValue()` will always return a string anyway, so the `|| unknownValue()` will never get called.

Please update this for all occurrences.